### PR TITLE
Pipeline + queries A21

### PR DESCRIPTION
**Pull request by cmdoret**
This PR does 2 things:
* Add dataset A21 (https://github.com/statistikZH/statbotData/issues/72) from Basel Landschaft "Endverbrauch von Elektrizität nach Gemeinde und Jahr (seit 1990)". Includes pipeline, metadata and 8 queries (in german)
* Implements a municipality-level spatialunit mapper (A21 was used as a test bed to develop it). It requires temporal information as in put and works with:
  + Duplicate names in different cantons
  + Renamed / Merged / Split municipalities
  + Municipalities that changed cantons

### CODE REVIEW

**Comment by nooralahzadeh** [comment]
Hi @cmdoret ,
I cannot find the queries for A21. Please let me know where they are located.
Thanks

File: pipelines/A21/queries.sql
Comment (nooralahzadeh) [inline pipelines/A21/queries.sql:1]: Hi @cmdoret 
I think the following minor changes would be nice to apply to the NL questions:

Query N1. 1: 
Which municipality in Basel-Landschaft had the highest electricity consumption in 1995?

Query Nr. 2: 
Show me the total electricity consumption of municipalities in Basel-Landschaft in 2000 vs. 2020.

Query Nr. 3:  
In 2012, which three municipalities in Basel-Landschaft had the lowest electricity consumption?

Query Nr. 6: 
In 2018, among Bottmingen and Birsfelden municipalities in Basel-Landschaft, which one had the highest electricity consumption?

Query Nr. 7: 
In which year did Muttenz in Basel-Landschaft have its highest electricity consumption?

Query Nr. 8: 
What was the peak annual electricity consumption ever recorded for a municipality in Basel-Landschaft? Show also the municipality name.

File: pipelines/A21/queries.log
Comment (nooralahzadeh) [inline pipelines/A21/queries.log:1]: Hi @cmdoret 
I think the following minor changes would be nice to apply to the NL questions:

Query N1. 1: 
Which municipality in Basel-Landschaft had the highest electricity consumption in 1995?

Query Nr. 2: 
Show me the total electricity consumption of municipalities in Basel-Landschaft in 2000 vs. 2020.

Query Nr. 3:  
In 2012, which three municipalities in Basel-Landschaft had the lowest electricity consumption?

Query Nr. 6: 
In 2018, among Bottmingen and Birsfelden municipalities in Basel-Landschaft, which one had the highest electricity consumption?

Query Nr. 7: 
In which year did Muttenz in Basel-Landschaft have its highest electricity consumption?

Query Nr. 8: 
What was the peak annual electricity consumption ever recorded for a municipality in Basel-Landschaft? Show also the municipality name.

**Review by nooralahzadeh — COMMENTED** [review]
Hi @cmdoret 
I think the following minor changes would be nice to apply to the NL questions:

Query N1. 1: 
Which municipality in Basel-Landschaft had the highest electricity consumption in 1995?

Query Nr. 2: 
Show me the total electricity consumption of municipalities in Basel-Landschaft in 2000 vs. 2020.

Query Nr. 3:  
In 2012, which three municipalities in Basel-Landschaft had the lowest electricity consumption?

Query Nr. 6: 
In 2018, among Bottmingen and Birsfelden municipalities in Basel-Landschaft, which one had the highest electricity consumption?

Query Nr. 7: 
In which year did Muttenz in Basel-Landschaft have its highest electricity consumption?

Query Nr. 8: 
What was the peak annual electricity consumption ever recorded for a municipality in Basel-Landschaft? Show also the municipality name.

**Review by nooralahzadeh — APPROVED** [review]
Hi @cmdoret
I think the following minor changes would be nice to apply to the NL questions:

Query N1. 1:
Which municipality in Basel-Landschaft had the highest electricity consumption in 1995?

Query Nr. 2:
Show me the total electricity consumption of municipalities in Basel-Landschaft in 2000 vs. 2020.

Query Nr. 3:
In 2012, which three municipalities in Basel-Landschaft had the lowest electricity consumption?

Query Nr. 6:
In 2018, among Bottmingen and Birsfelden municipalities in Basel-Landschaft, which one had the highest electricity consumption?

Query Nr. 7:
In which year did Muttenz in Basel-Landschaft have its highest electricity consumption?

Query Nr. 8:
What was the peak annual electricity consumption ever recorded for a municipality in Basel-Landschaft? Show also the municipality name.

**Comment by cmdoret** [comment]
Thanks @nooralahzadeh  ! I applied your suggestions

**Pull request merged**